WORKSPACE

```diff
@@ -33,18 +33,18 @@ swift_rules_extra_dependencies()
 
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "d1ffd055969c8f8d431e2d439813e42326961d0942bdf734d2c95dc30c369566",
+    sha256 = "16e9fca53ed6bd4ff4ad76facc9b7b651a89db1689a2877d6fd7b82aa824e366",
```

Comment:
What is this sha pointing to? I went to the URLs listed below and they don't agree with this one.

Reply:
the SHA is from the official download page: https://github.com/bazelbuild/rules_go/releases/tag/v0.34.0

also the integration test passed, so this SHA has to be correct, no?

Reply:
I see, it is in the comments of that release. I was just trying to append the sha on those links below but nothing was discoverable.